env.example

```diff
@@ -0,0 +1,42 @@
+# Note that not all variables here might be in use for your selected configuration
+
+# When adding additional env variables, the schema in /env/schema.mjs should be updated accordingly
+
+# App
+
+NEXT_PUBLIC_APP_NAME="Senchabot"
```

Comment:
It would be good we remove nextjs related variables from the env.example in the main directory, Thanks!